unity/Holo/Assets/Editor/ModelPreprocessing/AssetBundles/AssetBundleCreator.cs

```diff
@@ -54,7 +54,7 @@ private void SaveAssetBundles()
         {
             AssetDatabase.CreateFolder("Assets", "StreamingAssets");
         }
-        BuildPipeline.BuildAssetBundles(Application.dataPath + "/StreamingAssets", buildMapArray, BuildAssetBundleOptions.None, BuildTarget.StandaloneWindows);
+        BuildPipeline.BuildAssetBundles(Application.dataPath + "/StreamingAssets", buildMapArray, BuildAssetBundleOptions.None, BuildTarget.WSAPlayer);
         AssetDatabase.DeleteAsset("Assets/StreamingAssets/StreamingAssets");

```

Comment:
Since we have trouble on hololens reading them, let''s try to add some options :) Instead of `BuildAssetBundleOptions.None`, try 
```
BuildAssetBundleOptions.UncompressedAssetBundle | 
BuildAssetBundleOptions.DeterministicAssetBundle |
BuildAssetBundleOptions.ForceRebuildAssetBundle |
BuildAssetBundleOptions.StrictMode
```

(these should be harmless, and may allow us to catch something?)

https://docs.unity3d.com/ScriptReference/BuildAssetBundleOptions.html

Reply:
Problems solved :) this changes are not needed.